packages/next-urql/README.md

```diff
@@ -0,0 +1,226 @@
+<div align="center">
+  <img src="assets/next_logo.png" alt="NextJS" height="200" width="200">
+  <img src="assets/urql_logo.png" alt="urql" height="200" width="200">
+  <br />
+  <a href="https://npmjs.com/package/next-urql">
+    <img alt="NPM Version" src="https://img.shields.io/npm/v/next-urql.svg" />
+  </a>
+  <a href="https://travis-ci.com/FormidableLabs/next-urql">
+    <img alt="Test Status" src="https://api.travis-ci.com/FormidableLabs/next-urql.svg?branch=master" />
+  </a>
+</div>
+
+## `next-urql`
+
+A set of convenience utilities for using `urql` with NextJS.
+
+### Motivation
+
+Using GraphQL with server-side rendering in React is a challenging problem. Currently, React has no support for `Suspense` for data fetching on the server. To get around this, a prepass step can be used to walk the tree (or a subsection of the tree) of your React application and suspend when it encounters thrown `Promise`s. For more information, check out [`react-ssr-prepass`](https://github.com/FormidableLabs/react-ssr-prepass).
+
+`next-urql` handles integrating this prepass step for you, such that your NextJS application using `urql` will prefetch your GraphQL queries on the server before sending down markup to the Client.
+
+### Installation
+
+Install `next-urql` along with its `peerDependencies`.
+
+```sh
+npm install --save next-urql urql react-is isomorphic-unfetch
+```
+
+`react-is`, `styled-components`, and `isomorphic-unfetch` help to support server-side `Suspense` with `react-ssr-prepass`. This assumes you have followed the basic installation steps for `urql` [here](https://github.com/FormidableLabs/urql#installation).
```

Comment:
is `styled-components`  needed since it's not in the above install

Reply:
Good point! That’s not required anymore

Reply:
Good catch, will remove it. It used to be needed in previous versions of `react-ssr-prepass`, but no longer!